person.rb

```diff
@@ -0,0 +1,27 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @parent_permission = parent_permission
+    @age = age
+  end
+  # Getters and setters
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def can_use_services?
+    of_age? || @parent_permission == true
+  end
+
+  # Private methods
+
+  private
+
+  def of_age?
+    if @age >= 18
+      returns true
+    else
+      returns false
+    end
```

Comment:
- [ ] [OPTIONAL] These lines could also be further shortened as follows ```@age >= 18```. The method will only return true if age is greater than or equal to 18 (Which is exactly what you want to achieve).

```
def is_of_age?
    @age >= 18
  end
```